Messaging.md

```diff
@@ -88,30 +86,18 @@ The following DLC-specific types are used throughout the specification. All type
 
 This type contains information about a contracts outcomes and their corresponding payouts. To save space, only one side's POV is included in this message as the other can be derived using `remote_payout = total_collateral - local_payout`.
 
-#### Version 0 `contract_info`
+#### Version 0 `enum_contract_info`
 
-1. type: 42768 (`contract_info_v0`)
+1. type: 42768 (`enum_contract_info_v0`)
 2. data:
-   * [`sha256`:`outcome_1`]
    * [`u64`:`outcome_1_local_payout`]
    * ...
-   * [`sha256`:`outcome_n`]
    * [`u64`:`outcome_n_local_payout`]
 
-This type of contract info is a simple enumeration where the value `n` is omitted from being explicitly included as it can be derived from the length field of the TLV.
-
-### The `oracle_info` Type
-
-This type contains information about the oracle(s) to be used in executing a DLC, and possibly the outcomes possible if these are not specified in the corresponding `contract_info`.
-
-#### Version 0 `oracle_info`
-
-1. type: 42770 (`oracle_info_v0`)
-2. data:
-   * [`x_point`:`oracle_public_key`]
-   * [`x_point`:`oracle_nonce`]
-
-This type of oracle info is for single-oracle, single signature (and hence single nonce) events.
+This type of contract info is a simple enumeration where the value `n` is omitted from being
+explicitly included as it can be derived from the length field of the TLV and should already be
+known from the corresponding `oracle_announcement`.
+The order of payouts matches the order of the outcomes in the `oracle_announcement`.
```

Comment:
The `oracle_announcement` doesn't have the outcomes, it is actually `oracle_announcement` -> `oracle_event` -> `event_descriptor`

Reply:
Maybe should exclude multi nonce descriptors here because I would have no idea what the ordering would be here

Reply:
not sure what you mean? If you find an enum_contract_info you better have gotten an enum_event_descriptor